solidity/ts/hookSalt.ts

```diff
@@ -1 +1 @@
-export const HOOK_SALT = 32171n
+export const HOOK_SALT = 242981n
```

Comment:
maybe this shouldn't be in the repo and just always run when building?

Reply:
Is it quick to generate, or does it take a while to grind for it?